Sources/Clappr/Classes/Base/BaseObject.swift

```diff
@@ -129,11 +129,15 @@ open class BaseObject: NSObject, EventProtocol {
 }
 
 public extension BaseObject {
-    public func trigger(_ event: Event) {
+    func trigger(_ event: Event) {
         trigger(event.rawValue)
     }
 
-    public func trigger(_ event: Event, userInfo: EventUserInfo) {
+    func trigger(_ event: Event, userInfo: EventUserInfo) {
         trigger(event.rawValue, userInfo: userInfo)
     }
+
+    func listenTo<T: EventProtocol>(_ contextObject: T, event: Event, callback: @escaping EventCallback) {
```

Comment:
Are we going to start replacing the calls on demand?

Reply:
What would you do, @igoriols ? :P I confess I have doubts. Maybe we can convert all of them on a separate PR?